README.md

```diff
@@ -19,15 +19,21 @@ Configuration
 
 It is required that a config.yaml exist in the root of the repository.  This will provide information for connecting to AWS.
 
+Note: If you are using Elasticsearch via AWS and your Elasticsearch domain is restricted use the last four lines of the example config to support IAM authentication. Otherwise emit the aforementioned config lines.
+
```

Comment:
"emit" may be misleading to a developer.  Could you instead just use the word "use"?